pandas/api/__init__.py

```diff
@@ -1,2 +1,4 @@
-""" public toolkit API """
+"""
```

Comment:
Does the docstring validator actually check module-level docstrings like this? Thought it was only class / function

Reply:
Reverted this.